internal/cmdrender/executor.go

```diff
@@ -425,6 +425,9 @@ func adjustRelPath(resources []*yaml.RNode, relPath string) ([]*yaml.RNode, erro
 				return resources, err
 			}
 		}
+		if strings.HasPrefix(path.Clean(currPath), "../") {
```

Comment:
couple of drive by comments:

shouldn't we do this check before updating the path annotation ? (above this code).

I am still noodling over another approach i.e. doing this in the `FunctionRunner` we have in pkg `fnruntime` instead of here. That's right after executing the function. I would like to enforce invariant as close as possible to the function execution.

is checking for `../` enough ?

Reply:
moved the logic to `fnruntime` package

also - I'm using path.Clean which should resolve things like `a/b/../../../c/d` to `../c/d`. Added a few unit tests for this.  Hopefully in this case checking for `../` is enough. I was trying to think of other ways that someone would be able to write outside of the package and none come to mind.